contracts/misc/NFTFloorOracle.sol

```diff
@@ -182,34 +170,18 @@ contract NFTFloorOracle is Initializable, AccessControl, INFTFloorOracle {
         _setupRole(UPDATER_ROLE, admin);
```

Comment:
admin may only need DEFAULT_ADMIN_ROLE because DEFAULT_ADMIN_ROLE hasRole(UPDATER_ROLE)

Reply:
for `DEFAULT_ADMIN_ROLE` we just accept price without execute any price check and aggregation logic, maybe useful in some  scenarios for price correction or emergency shutdown